docs/src/pages/components/slider/RangeSlider.js

```diff
@@ -14,13 +14,13 @@ export default function RangeSlider() {
   };
 
   return (
-    <Box sx={{ width: 300 }}>
+    <Box sx={{ width: 300, mt: 4 }}>
```

Comment:
Can you explain why you added this? This makes the component a bit off the center.

Reply:
![mui-slider](https://i.imgur.com/F1wRDPQ.png) I cant see the labels

Reply:
Inside codesandbox, yes, but this puts the demo example on docs page off the center. We want to keep consistency between examples, so I'd leave it as it is for now.

Reply:
> Inside codesandbox, yes, but it looks fine on docs page. This changes places the demo example on docs page off the center. We want to keep consistency between examples, so I'd leave it as it is for now.

 I understand your point of view, but as for me functionality is much more important than design. It may be possible to add "mt: "10%" instead?